modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -173,12 +149,13 @@ def __copy__(self):
         PandasOnRayDataframePartition
             A copy of this partition.
         """
+        if isinstance(self._data_ref, DeferredExecution):
+            self._data_ref.ref_count(1)
```

Comment:
we also make `data.ref_count(1)` in the partitions' constructor below, doesn't it mean that it creates two references instead of one unintentionally?

Reply:
Good catch! This ref increment code left from the previous versions. Removed.